pylintrc

```diff
@@ -18,6 +18,11 @@ extension-pkg-whitelist=
     gssapi,
     netifaces
 
+[CLASSES]
+
+# List of valid names for the first argument in a metaclass class method.
+valid-metaclass-classmethod-first-arg=mcs
```

Comment:
Have you tried to rename the first parameter of ``InnerClassMeta.__new__`` to ``cls``? Thta's the standard name for the first parameter to ``__new__``.

Reply:
That's odd, because [pylint documentation](http://pylint.pycqa.org/en/latest/technical_reference/features.html) says:

> __bad-mcs-classmethod-argument (C0204):__
> Metaclass class method %s should have %s as first argument Used when a metaclass class method has a first argument named differently than the value specified in valid-metaclass-classmethod-first-arg option (default to "mcs"), recommended to easily differentiate them from regular instance methods.

> __bad-mcs-method-argument (C0203):__
> Metaclass method %s should have %s as first argument Used when a metaclass method has a first argument named differently than the value specified in valid-classmethod-first-arg option (default to "cls"), recommended to easily differentiate them from regular instance methods.

And the linter output was:
```
ipapython/install/util.py:86: [C0204(bad-mcs-classmethod-argument),
    InnerClassMeta.__new__] Metaclass class method __new__ should have 'cls' as first argument)
```